src/components/MultiHopTrade/components/SharedTradeInput/SharedTradeInputFooter/SharedTradeInputFooter.tsx

```diff
@@ -126,16 +125,7 @@ export const SharedTradeInputFooter = ({
             invertRate={invertRate}
             noExpand={noExpand}
           >
-            <ReceiveSummary
-              isLoading={isLoading}
-              swapperName={swapperName}
-              swapSource={swapSource}
-              inputAmountUsd={inputAmountUsd}
-              affiliateBps={affiliateBps}
-              affiliateFeeAfterDiscountUserCurrency={affiliateFeeAfterDiscountUserCurrency}
```

Comment:
None of these was actually consumed in develop, see `<ReceiveSummary` in develop:

https://github.com/shapeshift/web/blob/61ce6cb82bd8077671d4854c34abaeea12c8c7fb/src/components/MultiHopTrade/components/SharedTradeInput/SharedTradeInputFooter/components/ReceiveSummary.tsx#L8-L26